File: C/08/Sample5.c
#include <stdio.h>

/* buy関数の定義 */
void buy(int x, int y)
{
  printf("%d万円と%d万円の車を買いました。\n", x, y);

}
/* buy関数の呼び出し */
int main(void)
{
  int num1, num2;
  printf("1台目はいくらの車を買いますか？\n");
  scanf("%d", &num1);

  printf("2台目はいくらの車を買いますか？\n");
  scanf("%d", &num2);

  buy(num1,num2);
  // buy(num2,num1);

   return 0;
}

File: Python/07/Sample5.py
def sell(place, num1=10, num2=5):
    print(place, "支店で", num1+num2, "万円の販売が行われました。")
    print("num2= ", num2)

# sell(20)
sell("東京")
sell("東京", 20,5)
sell("東京", num2=20, num1=10)
